masterbase/lib.py

```diff
@@ -501,6 +501,8 @@ def list_demos_helper(engine: Engine, api_key: str, page_size: int, page_number:
         demo_sessions
     WHERE
         active = false
+    ORDER BY
```

Comment:
I wonder if IDs are already monotonically increasing as stored under the Postgres storage engine, rendering this redundant. Either way, it's more explicit, which I like.

Reply:
I would assume there is some default order but yeah id rather not take chances 😅